docs/core-concepts/router.md

```diff
@@ -0,0 +1,146 @@
+# Router
+The router is responsible for matching routes to urls and updating the browser history. It also provides a way to navigate to routes, access the current route, and provides some routing utilities. See [Router](/api/types/Router.md) type for more information.
+
+## Creating a Router
+Create a router using the `createRouter` utility.
+
+```ts {8}
+import { createRoute, createRouter } from '@kitbag/router'
+
+const routes = [
+  createRoute(...),
+  createRoute(...),
+] as const
+
+const router = createRouter(routes)
+```
+
+## Installing
+Once you have created a router you can install it into a Vue application using the `use` method.
+
+```ts {8}
+import { createRouter } from '@kitbag/router'
+import { createApp } from 'vue'
+import App from './App.vue'
+
+const router = createRouter(...)
+const app = createApp(App)
+
+app.use(router)
+app.mount('#app')
+```
+
+## Type Safety
```

Comment:
is it worth repeating this from quick start?

Reply:
Lets discuss this in general on this comment https://github.com/kitbagjs/router/pull/395#discussion_r1899187620